pulp_docker/app/tasks/tag.py

```diff
@@ -0,0 +1,39 @@
+from pulpcore.plugin.models import Repository, RepositoryVersion, ContentArtifact
+from pulp_docker.app.models import ManifestTag, Manifest
+
+
+def create_new_repository_version(manifest_pk, tag, repository_pk):
```

Comment:
what about renaming it some something like 'tag_image', and 'untag_image'

Reply:
Do you mean renaming the file or the method? The method is responsible for creating a new repository which will contain a new tag.

Reply:
i meant to rename the method, i looked at examples like https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/app/tasks/copy.py#L6
https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/app/tasks/upload.py#L13
if you look, all of them create a new repo version as a result of copy/one-shot upload. in our case it will be tag/untag